BackfillPowerBI.yml

```diff
@@ -46,3 +74,11 @@ jobs:
       }
     name: Backfill
     displayName: Backfill PowerBI
+
+  - task: PublishPipelineArtifact@1
```

Comment:
purpose? whart artifacts would there be?

Reply:
We are capturing the report before we delete it.